web/docs/language/features.md

```diff
@@ -1649,29 +1649,20 @@ Default database is `SQLite`, since it is great for getting started with a new p
 Check below for more details on how to migrate from SQLite to PostgreSQL.
 
 ### PostgreSQL
-When using `PostgreSQL` as your database (`app: { db: { system: PostgreSQL } }`), you will need to spin up a postgres database on your own so it runs during development (when running `wasp start` or doing `wasp db ...` commands) and you will need to provide Wasp with `DATABASE_URL` environment variable that Wasp will use to connect to it.
+When using `PostgreSQL` as your database (`app: { db: { system: PostgreSQL } }`), you will need to make sure you have a postgres database running during development (when running `wasp start` or doing `wasp db ...` commands).
 
-One of the easiest ways to run a PostgreSQL database on your own is by spinning up [postgres docker](https://hub.docker.com/_/postgres) container when you need it with the following shell command:
-```
-docker run \
-  --rm \
-  --publish 5432:5432 \
-  -v my-app-data:/var/lib/postgresql/data \
-  --env POSTGRES_PASSWORD=devpass1234 \
-  postgres
-```
+To help with this, Wasp provides `wasp start db` that starts the default db for you. Your Wasp app will automatically connect to it once you have it running via `wasp start db`, no additional configuration is needed. Wasp will require that you have `docker` installed for this though.
```

Comment:
```suggestion
To help with this, Wasp provides `wasp start db` that starts the default db for you. Your Wasp app will automatically connect to it once you have it running via `wasp start db`, no additional configuration is needed. This command relies on Docker being installed on your machine.
```

Reply:
Sure, done! I am curious, why do you think this is better?